packages/button/src/utils/index.ts

```diff
@@ -1 +1,9 @@
 export * from './getStyles';
+
+
+export const getElement = (disabled: boolean, href?: string): string => {
+  if (href) {
+    return disabled ? 'span' : 'a';
```

Comment:
Why disabled param affects element type? It is strange logic, isn't it?

Reply:
Because tag a haven't prop disabled. https://www.w3schools.com/tags/tag_a.asp